mlir/include/mlir/Interfaces/TilingInterface.td

```diff
@@ -360,6 +360,43 @@ def TilingInterface : OpInterface<"TilingInterface"> {
         /*defaultImplementation=*/[{
           return failure();
         }]
+      >,
+      //===------------------------------------------------------------------===//
+      // Interface methods for querying fusability.
+      //===------------------------------------------------------------------===//
+      InterfaceMethod<
+        /*desc=*/[{
+          Indicates whether it is possible to fuse this operation with the given
+          result slice. This method is not allowed to generate any IR.
+        }],
+        /*retTy=*/"bool",
+        /*methodName=*/"isOpFusableWithConsumerSlice",
+        /*args=*/(ins
+          "unsigned":$resultNumber,
+          "::mlir::ArrayRef<::mlir::OpFoldResult>":$offsets,
```

Comment:
I think the consumer slice API has been modified to handle multiple results at the same time. I think this needs to do the same.

Reply:
`getIterationDomainTileFromResultTile` still only looks at a single result at a time, so this does too. If producing multiple results via a single tiling call is implemented I would expect the existing method to be updated to reflect that too, so I'd rather keep them consistent right now.

Reply:
Consumer is in reference to the slice here, so

This:
tilableOp -> consumer

Consumer Fusion:
LoopOp -> consumer tilable op

Consumer is unsurprisingly overloaded. I'm open to better naming suggestions if you have one (ditto below).